src/marqo/README.md

```diff
@@ -135,6 +149,28 @@ $ sudo apt-get install -y nvidia-docker2
 ```
 Once this is installed, one of the previous Docker commands can be run (either step B., C., or D.).
 
+### Using Marqo on an AWS EC2 machine
+#### (Note: This is not recommended for production use cases.)
+
+1. Install docker
+
+To install Docker (through terminal) go to the [Official Docker Website](https://docs.docker.com/engine/install/ubuntu/)
+
+2. Set up SSH Config (to stop timeouts)
+
+Edit the SSH config file with `nano ~/.ssh/config` then insert the line: `ServerAliveInterval 50`
+
+3. Run marqo-os
+```
+sudo docker rm -f marqo-os; sudo docker run -p 9200:9200 -p 9600:9600 -e "discovery.type=single-node" marqoai/marqo-os:0.0.2-arm
```

Comment:
nit: you'd have to ensure you make an Arm Ec2 instance (which I think is the default).